WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,23 @@
 RSpec.configure do |config|
+  reference_tables_to_keep = %w(Countries Continents Events Rounds Formats teams)
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: reference_tables_to_keep)
+    reference_tables_to_keep.each do |table|
+      ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+      load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
This still looks quite gross to me. Executing code by `load`, and using `db/seeds` in unintended ways.

I think this should be regular code, perhaps living in `spec/support/PopulateTestDb.rb`.